git_machete/client.py

```diff
@@ -1950,25 +1950,28 @@ def __get_valid_pull_requests(prs_list: Optional[List[int]],
                         if len(prs_list) > 1:
                             warn(f"PR #{pr_no} is not found in repository `{org}/{repo}`, skipping.")
                         else:
-                            raise MacheteException(f"PR #{pr_no} is not found in repository `{org}/{repo}`")
+                            warn(f"PR #{pr_no} is not found in repository `{org}/{repo}`")
```

Comment:
For this case, I'd leave an error... a user expected a specific PR number to be present, and this particular PR does not exist

Reply:
As opposed to the other cases, where zero open PRs are a kinda expected/possible case